app/controllers/admin/merchants_controller.rb

```diff
@@ -1,19 +1,34 @@
 class Admin::MerchantsController < Admin::BaseController
 
-    def index
-      @merchants = Merchant.all
-    end
 
-    def show
-        @merchant = Merchant.find(params[:id])
-    end
+  def index
+    @merchants = Merchant.all
+  end
 
-    def update
-      merchant = Merchant.find(params[:id])
+  def update
```

Comment:
nice refactor combining both enable and disable

Reply:
I agree with Melanie. I like this combining of of the two methods